.github/workflows/automerge.yml

```
@@ -1,44 +1,196 @@
-          intervalSeconds: 30
 
-      - name: Check CI result
-        if: steps.wait-for-ci.outputs.conclusion != 'success'
+      - name: Configure git
+        run: |
+          git config user.name "github-actions[bot]"
+          git config user.email "github-actions[bot]@users.noreply.github.com"
+
+      - name: Try rebase onto base branch
+        id: rebase
         run: |
```

actionlint (1.7.10)

[error] "github.event.pull_request.head.ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)